### PR TITLE
add parameter for indicating an array of actions not to buffer

### PR DESCRIPTION
**Pull request by itslittlejohn**
I think this should address issue #14. I also added a very similar test to show what my changes would do. 

Also, would this really constitute a breaking change if we only are adding an additional parameter? The first test passed without any modifications so in that sense it wouldn't effect current consumers.

Just let me know what you think!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,38 +2,39 @@ var BUFFERED_ACTION_RETURN = 'redux-action-buffer: buffered action'
 
 var setImmediate = typeof global !== 'undefined' && typeof global.setImmediate !== 'undefined' ? global.setImmediate : setTimeout
 
-module.exports = function bufferActions (breaker, cb) {
+module.exports = function bufferActions(breaker, cb, passthrough) {
```
Comment (rt2zz) [inline index.js]: how about this api instead:
`bufferActions(options, cb)`
where options is an object with `{ breaker, passthrough }`.

This way we can keep backwards compat via mapping the arguments if a string is provided in the first argument. Also keeps callback as final arg which is idiomatic.

**Comment by rt2zz** [comment]
i.e.
```js
module.exports = function bufferActions(options, cb) {
  var active = true
  var queue = []
  var passthrough = options && options.passthrough || []
   
  var breaker = typeof options === 'object'
    ? options.breaker
    : options

  var breakerType = typeof breaker
  //...
}
```

Kind of messy but it should work. That or we can just make a breaking change and release a new major version.

**Comment by itslittlejohn** [comment]
Yeah, I like that better too. I'll clean it up in the morning.

**Comment by itslittlejohn** [comment]
Changes made, and I also updated the readme to reflect the new argument definition. Feel free to tweak it. Tests still pass so I think we're good?

**Comment by itslittlejohn** [comment]
Awesome! Thanks. Also, I realized I forgot to bump the minor version in package.json. Could you do that and push a new package to npm when you get a chance?

**Pull request merged**